content/guides/component-testing/custom-mount-vue.md

```diff
@@ -270,6 +247,63 @@ components that use all of Vue's language features.
 
 Congrats! Happy building. 🎉
 
+### Vuetify
+
+<code-group>
+<code-block label="Vuetify 3" active>
+
+```js
```

Comment:
Hmmm,  I wonder what the policy/philosophy is around having specific examples, especially for non-mainline versions (eg, Vue 3 is the main one, this looks like docs for Vue 2).

I wonder if this might be better as a featured blog post? We can definitely facilitate working with community members on posts for https://www.cypress.io/blog.